### PR TITLE
symfony-cli: 4.26.9 -> 5.0.7

### PR DESCRIPTION
**Pull request by drupol**
The Symfony-cli tool has been [open-sourced](https://symfony.com/blog/announcing-the-open-sourcing-of-the-symfony-cli) so it's time to update the derivation.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by drupol** [comment]
@ofborg build

**Comment by drupol** [comment]
@ofborg build symfony-cli

**Comment by SebTM** [comment]
Tested and works fine for me 👍🏻

File: pkgs/development/tools/symfony-cli/default.nix [hunk 1]
```diff
@@ -1,50 +1,25 @@
...
 
   meta = with lib; {
     description = "Symfony CLI";
-    homepage = "https://symfony.com/download";
-    license = licenses.unfree;
+    homepage = "https://github.com/symfony-cli/symfony-cli";
+    license = licenses.agpl3Plus;
     maintainers = with maintainers; [ drupol ];
-    platforms = [ "x86_64-linux" "aarch64-linux" "i686-linux" "x86_64-darwin" ];
+    platforms = platforms.linux ++ platforms.darwin;
```
Comment (jtojnar) [inline pkgs/development/tools/symfony-cli/default.nix:23]: Does not Go set these by default?

Comment (drupol) [reply pkgs/development/tools/symfony-cli/default.nix:23]: How can I make sure of this ?

Comment (jtojnar) [reply pkgs/development/tools/symfony-cli/default.nix:23]: Delete it and check `symfony-cli.meta.platforms` in `nix repl`. Or maybe it is mentioned in Go docs or source code.

Comment (drupol) [reply pkgs/development/tools/symfony-cli/default.nix:23]:
```
$ nix repl '<nixpkgs>'                                                                                                                                                                    
Welcome to Nix 2.5.1. Type :? for help.

Loading '<nixpkgs>'...
Added 15639 variables.

nix-repl> :lf .
warning: Git tree '/home/pol/dev/git/nixpkgs' is dirty
Added 13 variables.

nix-repl> symfony-cli.meta.platforms
[ "x86_64-linux" "aarch64-linux" "i686-linux" "x86_64-darwin" ]

nix-repl>
```

Comment (drupol) [reply pkgs/development/tools/symfony-cli/default.nix:23]: Going to remove them.

**Comment by drupol** [comment]
@ofborg build symfony-cli

**Comment by drupol** [comment]
@ofborg build symfony-cli

**Comment by drupol** [comment]
Cool, thanks @jtojnar !

**Pull request merged**